CONVENTIONS.md

```diff
@@ -15,6 +16,12 @@
   - [Identifiers](#identifiers)
   - [Variables](#variables)
 
+## Plugin Location
```

Comment:
Should this be "Creating New Plugins"?

Reply:
Modifying this gives the title room to expand for future issues if that is what you mean.